dace/frontend/python/replacements.py

```diff
@@ -692,6 +692,14 @@ def _log(pv: ProgramVisitor, sdfg: SDFG, state: SDFGState, input: str):
     return _simple_call(sdfg, state, input, 'log')
 
 
+@oprepo.replaces('log10')
+@oprepo.replaces('dace.log10')
+@oprepo.replaces('numpy.log10')
```

Comment:
AFAIK the NumPy log10 ufunc is already supported through the Python frontend's ufunc interface (see `dace/frontend/python/replacements.py`). Could you confirm this and remove the above line so it doesn't interfere with calls that, e.g., use ufunc keyword arguments (any supported by DaCe)?

Reply:
I can remove this line but I haven't been able to confirm that `numpy.log10` works